src/js/components/CharityPage.jsx

```diff
@@ -44,6 +45,18 @@ const CharityPage = () => {
 	if ( ! charity) {
 		return <Misc.Loading pv={pvCharity} />;
 	}
+	let impact;
+	let project = NGO.getProject(charity);
+	if (project) {
+		let targetCount = DataStore.getValue([
+			"widget",
+			"SearchResults",
+			cid,
+			"targetCount",
+		]);
+		const outputs = Project.outputs(project);
+		impact = impactCalc({ charity, project, output:outputs[0], targetCount: targetCount || 1 });
+	}
```

Comment:
How about moving this code *inside* ImpactCalculatorSession, and just returning an empty div or something if impact is false. I think that would be neater because it keeps the impact-calculator logic a bit more self-contained. (Could even consider putting this code inside an exported method of ImpactCalculator.jsx but that's probably unnecessary).

Reply:
OK